drip/utils.py

```diff
@@ -142,7 +142,16 @@ def give_model_field(full_field, Model):
 
 
 def get_simple_fields(Model, **kwargs):
-    return [[f[0], f[3].__name__] for f in get_fields(Model, **kwargs)]
+    ret_list = []
+    for f in get_fields(Model, **kwargs):
+        if '__' in f[0]:
+            # Add __user__ to the fields in related models
+            parts = f[0].split('__')
+            f[0] = parts[0] + '__user__' + parts[1]
```

Comment:
```suggestion
            f[0] = f'{parts[0]}__user__{parts[1]}'
```

Reply:
I did this at first, but in travis when you test that using python 3.5, an error is thrown. So to be compatible with python 3.5 and higher, I need the `+` operation

Reply:
I think you could use `format`.
```suggestion
            f[0] = '{}__user__{}'.format(parts[0], parts[1])
```

Reply:
I will merge this and make an issue to review the string format in this project